examples/example-hostname/README.md

```diff
@@ -0,0 +1,60 @@
+gRPC Hostname Example
+=====================
+
+The hostname example is a Hello World server whose response includes its
+hostname. It also supports health and reflection services. This makes it a good
+server to test infrastructure, like load balancing.
+
+The example require grpc-java to already be built. You are strongly encouraged
```

Comment:
s/require/requires/

Reply:
Nobody noticed this for any other example!